ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -64,40 +67,37 @@
   private final DocumentClient documentClient;
   private final String cosmosCollectionLink; // eg "/dbs/ambry-metadata/colls/blob-metadata"
   private final RequestOptions defaultRequestOptions = new RequestOptions();
+  private final OperationContext blobOpContext = new OperationContext();
   private final AzureMetrics azureMetrics;
 
-  // TODO: these constants should live somewhere else, but where?
-  private static final String HTTP = "http";
-  private static final String HTTPS = "https";
-  private static final String PROXY_HOST = "proxyHost";
-  private static final String PROXY_PORT = "proxyPort";
-
   /**
    * Construct an Azure cloud destination from config properties.
+   * @param cloudConfig the {@link CloudConfig} to use.
    * @param azureCloudConfig the {@link AzureCloudConfig} to use.
    * @param azureMetrics the {@link AzureMetrics} to use.
    * @throws InvalidKeyException if credentials in the connection string contain an invalid key.
    * @throws URISyntaxException if the connection string specifies an invalid URI.
    */
-  AzureCloudDestination(AzureCloudConfig azureCloudConfig, AzureMetrics azureMetrics)
+  AzureCloudDestination(CloudConfig cloudConfig, AzureCloudConfig azureCloudConfig, AzureMetrics azureMetrics)
       throws URISyntaxException, InvalidKeyException {
     this.azureMetrics = azureMetrics;
-    azureAccount = CloudStorageAccount.parse(azureCloudConfig.storageConnectionString);
+    azureAccount = CloudStorageAccount.parse(azureCloudConfig.azureStorageConnectionString);
     azureBlobClient = azureAccount.createCloudBlobClient();
+    // Check for proxy
+    if (cloudConfig.vcrProxyHost != null) {
+      logger.info("Using proxy: {}:{}", cloudConfig.vcrProxyHost, cloudConfig.vcrProxyPort);
```

Comment:
`AzureCloudDestination using proxy:.....`

Reply:
The class name is printed automatically by log4j before the message.